src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -351,10 +358,12 @@ private void contributeInstanceItems(UUID centralServerId, Instance instance, St
   }
 
   private void contributeItem(UUID centralServerId, String bibId, Item item, Statistics stats) {
+    log.info("contributeItem:: parameters centralServerId: {}, bibId: {}, item id: {}, stats: {}", centralServerId, bibId, item.getId(), stats);
     contributeItemsChunk(centralServerId, bibId, List.of(item), stats);
   }
 
   private void contributeItemsChunk(UUID centralServerId, String bibId, List<Item> items, Statistics stats) {
+    log.info("contributeItemsChunk:: paramerters centralServerId: {}, bibId: {}, items: {}, stats: {}", centralServerId, bibId, items, stats);
```

Comment:
Do we want to log the Items list? I doubt that it will be converted to a meaningful string. Even if it is, it will be too much for the logs.

Reply:
Maybe just log the count of the items list.

Reply:
Added items count in the log.